api/httputil/secure_http_dump.go

```diff
@@ -9,14 +9,13 @@ import (
 	"net/http/httputil"
 	"regexp"
 	"time"
-
-	"github.com/goware/prefixer"
 )
 
 const secureMask = "******"
 
 var (
-	tokenRe = regexp.MustCompile(`"((?i)access_token|(?i)refresh_token)":\s*?".*?"`)
+	tokenRe           = regexp.MustCompile(`"((?i)access_token|(?i)refresh_token)":\s*?".*?"`)
+	beginningOfLineRe = regexp.MustCompile("(?m)^")
```

Comment:
Just for consistency with other projects
```suggestion
	tokenRegex           = regexp.MustCompile(`"((?i)access_token|(?i)refresh_token)":\s*?".*?"`)
	beginningOfLineRegex = regexp.MustCompile("(?m)^")
```